systemd/jenkins.service

```diff
@@ -0,0 +1,142 @@
+#
+# This file is managed by systemd(1). Do NOT edit this file manually!
+# To override these settings, run:
+#
+#     systemctl edit @@ARTIFACTNAME@@
+#
+# For more information about drop-in files, see:
+#
+#     https://www.freedesktop.org/software/systemd/man/systemd.unit.html
+#
+
+[Unit]
+Description=Jenkins Continuous Integration Server
+Requires=network.target
+After=network.target
```

Comment:
Better to use `After=network-online.target` to be sure that the network is ready

Reply:
> Better to use `After=network-online.target` to be sure that the network is ready

Possibly.

Reply:
Well, looks like that despite the fact you are requesting review and tedting you're not welcome to any reasonable suggestions, so why are you requesting them?:)

Reply:
> Well, looks like that despite the fact you are requesting review and tedting you're not welcome to any reasonable suggestions, so why are you requesting them?:)

Unless it's wrong at this stage testing is more important 😄

Reply:
What I can see from the PR that I still have to fully override service configuration from the default package by configuration management software the same way that I did for the last 6 years starting from 2016, so what the point to add weak code?=)

Reply:
I do agree that I'm a bad person because I used systemd units for Jenkins for 6 years at least but didn't provide a PR with that but at this point you got people attention and don't want to use their experience:)
Looks strange to me.

Reply:
Feel free to submit a PR afterwards, to make these minor changes. 

This is an initial v1 for systemd, not frozen in time.